src/mono/mono/metadata/native-library.c

```diff
@@ -1256,6 +1260,20 @@ legacy_lookup_native_library (MonoImage *image, const char *scope)
 
 #endif // ENABLE_NETCORE
 
+#if defined(ENABLE_NETCORE) && !defined(NO_GLOBALIZATION_SHIM)
+static gpointer
+default_resolve_dllimport(const char* dll, const char* func)
+{
+	if (strcmp(dll, "libSystem.Globalization.Native") == 0) {
+		const void* method_impl = GlobalizationResolveDllImport(func);
```

Comment:
```suggestion
		const void *method_impl = GlobalizationResolveDllImport (func);
```
I think these changes as a whole should match the Mono coding style within Mono files. This is an example of one such stylistic change. The Mono style is admittedly a bit odd and different from CoreCLR, but it would be nice to keep it consistent.

Reply:
Tried my best to pick the style of containing code. :-)  
Some nuances are hard to notice though.

Reply:
Gotcha. The main things I see are the two changes I showed here: the asterisk goes on the right and we put a space before the argument list for a function (and before array indices, sadly).

For another example: `default_resolve_dllimport(const char* dll, const char* func)` -> `default_resolve_dllimport (const char *dll, const char *func)`.

Reply:
If you'd prefer I can just propose the rest of the style fixes as changes on GH. Whatever works :)

Reply:
I would probably miss something anyways.